sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/auth/SimpleWebAuthServiceImpl.java

```diff
@@ -25,8 +26,9 @@
  * @author cdfive
  * @since 1.6.0
  */
-@Primary
 @Component
+@Primary
+@ConditionalOnProperty(name = "auth.enabled", matchIfMissing = true)
```

Comment:
Suggest remain `havingValue = "true"` , since if remove it, we can set `auth.enabled` to any expression, this is not clear semantic understanding.

In `FakeAuthServiceImpl`, `havingValue = "false"`, without `matchIfMissing = true`.

Reply:
Conversely, if we add `havingValue = "true"`, auth.enabled is any expression except `"true"`, the Sentinel login will be disable 🤣

> To make the UT `GatewayApiControllerTest` passed, remove the `@ConditionalOnProperty`
> in `FakeAuthServiceImpl` is okay.

I thought so

Reply:
> Conversely, if we add havingValue = "true", auth.enabled is any expression except "true", the Sentinel login will be disable
Agree with you, it's okay and easy use for users.